@here/harp-mapview/lib/text/TextElementState.ts

```diff
@@ -176,6 +177,7 @@ export class TextElementState {
             this.element.bounds = predecessor.element.bounds;
             this.element.glyphCaseArray = predecessor.element.glyphCaseArray;
         }
+        this.element.textBufferObject = undefined;
```

Comment:
Here we have few options -> we may also:
```
 if (this.element.glyphs === undefined) {
            this.element.glyphs = predecessor.element.glyphs;
            ...
            this.element.textBufferObject = predecessor.element.textBufferObject;
        }
       else {
           this.element.textBufferObject = undefined;
       }
```
But definitely text buffer need to be changed here, cause we already retrieved new render and layout state.